components/brave_wallet/browser/json_rpc_service.h

```diff
@@ -399,13 +400,13 @@ class JsonRpcService : public mojom::JsonRpcService {
                               const std::string& error_message)>;
   void GetEthNftStandard(const std::string& contract_address,
                          const std::string& chain_id,
-                         const std::vector<std::string>& interfaces,
+                         base::span<const std::string_view> interfaces,
```

Comment:
I'm curious why we need const here? As std::string_view is immutable itself.

Reply:
It seems this `const` should come from generic way of typing a span over immutable elemets.